ProtoGame2D/Code/Game/Framework/Game.cpp

```diff
@@ -1,25 +1,103 @@
 #include "Game.h"
 
+#include "Engine/Core/GameCommon.h"
+#include "Engine/Graphics/Renderer.h"
+#include "Engine/Graphics/Shader.h"
+#include "Engine/Graphics/ShaderDefinition.h"
+#include "Engine/Graphics/Buffers/FrameBuffer.h"
+#include "Engine/Input/InputSystem.h"
+#include "Engine/Input/WinKeys.h"
+
+float x = 512.0f;
+
+extern InputSystem* g_InputSystem;
+
 Game::Game()
 {
+	m_ShaderDef = new ShaderDefinition(*ShaderDefinition::InitializeDef(APEX_SHADER_XML));
+	m_Shader = Renderer::GetInstance()->GetOrCreateShader(m_ShaderDef);
+	
+	m_ScreenShaderDef = new ShaderDefinition(*ShaderDefinition::InitializeDef(APEX_SCREENSHADER_XML));
+	m_ScreenShader = Renderer::GetInstance()->GetOrCreateShader(m_ScreenShaderDef);
+	
+	Renderer::GetInstance()->EnableBlend(ParseBlendFac[m_Shader->GetSRC()], ParseBlendFac[m_Shader->GetDST()], ParseBlendOp[m_Shader->GetOP()]);
```

Comment:
Render function call hence move to render function of the game

Reply:
Did we not have a conversation bout Renderer owning shader instances? Why are we newing off a ShaderDefinition object here?

Reply:
Done